packaging/ubuntu-16.04/changelog

```diff
@@ -1,8 +1,13 @@
 snapd (2.44.2) UNRELEASED; urgency=medium
 
+  [ Michael Vogt ]
   * placeholder changelog
 
- -- Michael Vogt <michael.vogt@ubuntu.com>  Thu, 02 Apr 2020 10:42:29 +0200
+  [ Zygmunt Krynicki ]
```

Comment:
No need to add anything to the changelog, it is auto-generated anyway from the PR title

Reply:
Removed